source/frontend/src/components/maps/MapLeafletView.tsx

```diff
@@ -274,10 +273,10 @@ const MapLeafletView: React.FC<React.PropsWithChildren<MapLeafletViewProps>> = (
         <LegendControl />
         <ZoomOutButton />
         <ScaleControl position="bottomleft" metric={true} imperial={false} />
-        <AdvancedFilterButton
+        {/* <AdvancedFilterButton
           onToggle={mapMachine.toggleMapFilterDisplay}
           active={mapMachine.isFiltering}
-        />
+        /> */}
```

Comment:
I'd rather remove the code instead of commenting it out - we have git history if we ever want to bring it back